app/views/partials/_author.html.erb

```diff
@@ -0,0 +1,14 @@
+<%if what_to_show == :avatar %>
```

Comment:
I'd replace the name of this local to be just `show`. `what_to_show` seems weird